platforms/documentation/docs/src/docs/userguide/dep-man/basics/declaring_dependencies.adoc

```diff
@@ -0,0 +1,227 @@
+
+These are known as file dependencies because they represent files without any <<dependency_management_terminology.adoc#sub:terminology_module_metadata,metadata>> (such as information about transitive dependencies, origin, or author) attached to them.
+
+image::dependency-management-file-dependencies.png[]
+
+To add files as dependencies for a configuration, you simply pass a <<working_with_files.adoc#sec:file_collections,file collection>> as a dependency:
+
+====
+include::sample[dir="snippets/artifacts/externalDependencies/kotlin",files="build.gradle.kts[tags=file-dependencies]"]
+include::sample[dir="snippets/artifacts/externalDependencies/groovy",files="build.gradle[tags=file-dependencies]"]
+====
+
+== Looking at an example
+
+Let's imagine an example for a Java application which uses link:https://github.com/google/guava[Guava], a set of core Java libraries from Google:
+
+image::declaring-dependencies-2.png[]
+
+The Java app contains the following Java class:
+
+[source, java]
+.InitializeCollection.java
+----
+package com.myapp;
+
+import com.google.common.collect.ImmutableMap;  // Comes from guava library
+import java.util.Map;
+
+public class InitializeCollection {
+    public static void main(String[] args) {
+        Map items = ImmutableMap.of("coin", 3, "glass", 4, "pencil", 1);
+    }
+}
+----
+
+To add the link:https://mvnrepository.com/artifact/com.google.guava/guava[Guava] library to your Gradle project as a dependency, you must add the following line to your build file:
+
+[.multi-language-sample]
+=====
+[source, kotlin]
+.build.gradle.kts
+----
+dependencies {
+    implementation("com.google.guava:guava:30.1.1-jre")
+}
+----
+=====
+[.multi-language-sample]
+=====
+[source,groovy]
+.build.gradle
+----
+dependencies {
+    implementation 'com.google.guava:guava:30.1.1-jre'
+}
+----
+=====
+
+Where:
+
+* `implementation` is the configuration.
+* `com.google.guava:guava:30.1.1-jre` specifies the group, name, and version of the library:
+** `com.google.guava` is the group ID.
+** `guava` is the artifact ID (i.e., name).
+** `30.1.1-jre` is the version.
+
+Take a quick look at the link:https://mvnrepository.com/artifact/com.google.guava/guava[Guava page in Maven Central] as a reference.
+
+[[sec:listing_dependencies]]
+== Listing project dependencies
+
+The `dependencies` task provides an overview of the dependencies of your project.
+It helps you understand what dependencies are being used, how they are resolved, and their relationships, including any transitive dependencies by rendering a dependency tree from the command line.
+
+This task can be particularly useful for debugging dependency issues, such as version conflicts or missing dependencies.
+
+For example, let's say our `app` project contains the follow lines in its build script:
+
+[source, kotlin]
+.app/build.gradle.kts
+----
+dependencies {
+    implementation("com.google.guava:guava:30.0-jre")
+    runtimeOnly("org.json:json:20220924")
```

Comment:
Can we pick a more common dependency, like [commons-lang3](https://mvnrepository.com/artifact/org.apache.commons/commons-lang3)

I think it makes sense to pick a few dependencies we think are popular and stick to using those in examples. 

Eventually (or ideally sooner rather than later), all of these examples should be snippets and we would want to avoid variation